tests/testem.js

```diff
@@ -38,8 +25,7 @@ fluid.defaults("fluid.tests.testem", {
         tests:   "%infusion/tests"
     },
     testemOptions: {
-        launch: "Chrome,Firefox",
```

Comment:
I tried to follow the comment thread above and saw that this change was made deliberately, but I can't quite reconstruct the reasoning - do we really want to prevent tests running in Firefox in local runs? This would prevent detection of a number of Firefox-only issues, e.g. the TextToSpeech tests which only run in Firefox because of a long-standing bug in the Chrome speech API.

Reply:
OK, I updated the setting to use a combination of `Headless Chrome` and `Firefox`.